src/credentials/attestation_verifier/FileAttestationTrustStore.cpp

```diff
@@ -104,9 +104,10 @@ CHIP_ERROR FileAttestationTrustStore::GetProductAttestationAuthorityCert(const B
     {
         uint8_t skidBuf[Crypto::kSubjectKeyIdentifierLength] = { 0 };
         MutableByteSpan candidateSkidSpan{ skidBuf };
-        VerifyOrReturnError(CHIP_NO_ERROR ==
-                                Crypto::ExtractSKIDFromX509Cert(ByteSpan{ candidate.data(), candidate.size() }, candidateSkidSpan),
-                            CHIP_ERROR_INTERNAL);
+        if (CHIP_NO_ERROR != Crypto::ExtractSKIDFromX509Cert(ByteSpan{ candidate.data(), candidate.size() }, candidateSkidSpan))
```

Comment:
No item in the trust store should fail to have a SKID. Why do you see an error happen here?

Reply:
We cannot guarantee that there are no other der files under the PAA trust store path. In my test, if the Certification Declaration file is in the PAA trust store path and it is read before the PAA cert file, the `GetProductAttestationAuthorityCert` will return ERROR before it gets the right PAA cert.

I think we should skip the der file if it does not have a SKID.

Reply:
I have created an issue #22331